app/views/events/index.html.haml

```diff
@@ -14,8 +17,17 @@
       %td 
         =link_to event.title, event_path(event), :class => 'link-underlined link-underlined-yellow'
       %td
-        =link_to 'Edit', edit_event_path(event), :class => 'btn btn-flat edit-event'
+        - if user_signed_in?
+          =link_to 'Edit', edit_event_path(event), :class => 'btn btn-flat edit-event'
+        - else
```

Comment:
Line 22 and 23  and 27 and 28 should be deleted. So no else anymore. The "if user_signed_in?" can be used only once if you include it above the two "%td"s like this: 
![image](https://user-images.githubusercontent.com/40898193/97783414-dd323980-1b97-11eb-98e8-cb9da63a4b00.png)